packages/env/README.md

```diff
@@ -350,6 +350,29 @@ wp> ^C
 ✔ Ran `wp shell` in 'tests-cli'. (in 16s 400ms)
 ```
 
+#### Installing a plugin or theme on the development instance
+
+```sh
+wp-env run cli plugin install custom-post-type-ui
+
+Creating 500cd328b649d63e882d5c4695871d04_cli_run ... done
+Installing Custom Post Type UI (1.9.2)
+Downloading installation package from https://downloads.wordpress.org/plugin/custom-post-type-ui.zip...
+The authenticity of custom-post-type-ui.zip could not be verified as no signature was found.
+Unpacking the package...
+Installing the plugin...
+Plugin installed successfully.
+Success: Installed 1 of 1 plugins.
+✔ Ran `plugin install custom-post-type-ui` in 'cli'. (in 6s 483ms)
+```
+
+**NOTE**: Depending on your host OS, you may experience errors when trying to install plugins or themes (e.g. `Warning: Could not create directory.). This is typically because the user ID used within the container does not have write access to the mounted directories created by `wp-env`. To resolve this, run the `docker-compose` command directly from the directory created by `wp-env` and add `-u $(id -u)` and `-e HOME=/tmp` the `run` command as options:
```

Comment:
I think backtick is missing after "`Warning: Could not create directory." and results in misformatted text in note section - [rendered version.](https://github.com/WordPress/gutenberg/blob/800f97028f2329763885f50717218e42fb8d9c5d/packages/env/README.md#installing-a-plugin-or-theme-on-the-development-instance)

Reply:
Gah, thanks for catching that! I've updated the page to include that backtick.